### ISSUE TITLE
Very Good Implementation based on Really Good Ideas, but Where is the Source Code?

### ISSUE THREAD

**Issue by johannhuang** [verysync/releases#1]
It seems like this is a project based on some similar open source project, is this a open source project? Or using github is just for issues?

**Comment by johannhuang** [comment]
By the way, for the later comers, the Souce Code downloadable file in release is not real Souce Code but just one nonsense README.md with only one line of content as `# Release`

**Comment by Catfriend1** [comment]
https://github.com/syncthing/syncthing

**Comment by fragtion** [comment]
Yes, we would like source code in order to develop translations into other languages (especially English), so as to widen application reach, popularity, and subsequent development.

Otherwise, if author chooses to keep source closed, how about offering translated product for us, please?

Regards

**Comment by Catfriend1** [comment]
The source code is there and it's publicly available.

**Comment by fragtion** [comment]
No, I am referring to the source code for VerySync specifically. Yes it appears to be influenced by `syncthing/syncthing` as you noted, but they are not the same project. We're on the VerySync issues page if you aren't paying attention ;)
